templates/edit-mod.tpl

```diff
@@ -55,125 +59,171 @@
+					Client and Server side mod</option>
 			</select>
 		</div>
-		
+
 		<div class="editbox">
 			<label>Logo/Thumbnail image</label>
 			<select name="logofileid" style="width:250px;">
 				{foreach from=$files item=file}
-					<option value="{$file['fileid']}"{if $asset['logofileid']==$file['fileid']} selected="selected"{/if}>{$file['filename']}</option>
+					<option value="{$file['fileid']}" {if $asset['logofileid']==$file['fileid']} selected="selected" {/if}>
+						{$file['filename']}</option>
 				{/foreach}
 			</select>
 		</div>
 
+		{if $asset['createdbyuserid'] == $user['userid']}
+			<div style="clear:both;"></div>
+			<h3>Ownership transfer</h3>
 
-		<!--<div style="clear:both;"></div>
-		<h3>Dependencies <a href="#addconnection" class="add" title="Add a connection"></a></h3>
-
-		<div class="connections">
-			{foreach from=$connections item=connection}
-				<div class="connection editbox" style="clear:both;">
-					<input type="hidden" name="connectionid[]" value="{$connection['connectionid']}">
-					<select name="connectiontypeid[]" class="required" style="width:150px;">
-						{foreach from=$connectiontypes item=connectiontype}
-							<option value="{$connectiontype['connectiontypeid']}" {if $connection['connectiontypeid'] == $connectiontype['connectiontypeid']}selected="selected"{/if}>{$connectiontype['name']}</option>
-						{/foreach}
-					</select>
-					
-					<select name="assettypeid[]" class="required" style="width: 150px">
-						<option value="">-</option>
-						{foreach from=$assettypes item=assettype}
-							<option value="{$assettype['assettypeid']}" {if $connection['asset']['assettypeid'] == $assettype['assettypeid']}selected="selected"{/if}>{$assettype['name']}</option>
-						{/foreach}
-					</select>
-
-					<select name="toassetid[]" class="required" style="width: 300px">
-						<option value="{$connection['asset']['assetid']}">{$connection['asset']['name']}</option>
-					</select>
-					
-					<a href="#" class="delete"></a>
-				</div>
-			{/foreach}
-		</div>
-		</form>-->
+			<div class="editbox linebreak">
+				{if isset($ownershipTransferUser) && $ownershipTransferUser}
+					<span>An ownership transfer invitation has been sent to: {$ownershipTransferUser}.</span>
+					<br>
+					<span>You may revoke the pending invitation using the button below:</span>
+					<p><a href="/edit/mod/?assetid={$asset['assetid']}&revokenewownership=1" class="button btndelete">REVOKE</a></p>
+				{else}
+					<div>
+						<label>Select new owner</label>
+						<small>Ownership can only be transferred by the current owner of this resource.</small>
+						<br>
+						<small>A notification will be sent to the specified user, inviting them to accept ownership.</small>
+						<br>
 
+						<select name="newownerid" style="width:300px;">
+							<option value="" selected="selected">--- Select new owner ---</option>
+							{foreach from=$users item=user}

```

Comment:
this overwrites the self user $user, needs to be renamed to `{foreach from=$users item=ouser}`